modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -85,7 +110,7 @@ Example DataWeave 2.0 expression to be used when configuring the policy:
 #[attributes.queryParams.'client_secret']
 ----
 
-== Credentials included in HTTP request payload
+==== With HTTP Request Payload
 Example request using curl:
 [source,text]
 ----
```

Comment:
It could also be possible to configure the policy to obtain the credentials from the request payload like in this example. Although feasible, it is not a recommended practice since it is harder to reflect in the API specification.

Reply:
Done!